types/changelog-parser/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for changelog-parser 2.7
+// Project: https://github.com/hypermodules/changelog-parser
+// Definitions by:  Adam Zerella <https://github.com/adamzerella>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+interface Options {
+    /**
+     * Path to changelog file
+     */
+    filePath: string;
+    /**
+     * Changelog file string to parse
+     */
+    removeMarkdown: boolean;
+}
+
+declare function parseChangelog(options: Options|string, callback?: (result: string, error?: string) => void): Promise<object>;
```

Comment:
this should probably be a `Partial<Options>` as you don't need to pass both options (you can pass `{ filePath: 'foo' }` for example).

Reply:
@43081j Good pickup, I've updated the types based on your feedback 😄